integration/app/lazy/detail/detail.module.ts

```diff
@@ -1,10 +1,11 @@
 import { NgModule } from '@angular/core';
-import { DetailComponent } from './detail.component';
 import { CommonModule } from '@angular/common';
 import { RouterModule } from '@angular/router';
 import { NgxsModule } from '@ngxs/store';
-import { DetailState } from './detail.state';
-import { routes } from './detail.routes';
+
+import { DetailComponent } from '@integration/lazy/detail/detail.component';
```

Comment:
```suggestion
import { DetailComponent } from './detail.component';
```
References to files within the same module should remain relative (see main review comment).

Reply:
actually using relative paths bad practice